pool.go

```diff
@@ -175,7 +175,8 @@ type Stat struct {
 	canceledAcquireCount  int64
 }
 
-// TotalResource returns the total number of resources.
+// TotalResource returns the total number of allocated resources in the pool.
+// The value is the sum of ConstructingResources, AcquiredResources, and IdleResources.
 func (s *Stat) TotalResources() int32 {
```

Comment:
I think using the word "allocated" could be confusing here. To me that is nearly a synonym for "acquired".